Technical Review: tshoot-packages.adoc (#737)
SUSE/doc-susemanager: modules/administration/pages/tshoot-packages.adoc

. Last step
////

> Packages on a client can sometimes be locked, which creates problems with metadata regeneration.

Reviewer: ```suggestion
When packages on a client are locked, {productname} Server may not be able to correctly determine the set of applicable patches.
```

Rationale: packages can be locked by users, they do not get locked because some process did. It is a known product limitation that the lock information is not considered by the Server when it determines the set of applicable patches.

When this occurs, package updates will be available in the {webui}, but will not appear on the client, and attempts to update the client will fail.
Check package locks and exclude lists to determine if packages are locked or excluded on the client.

> When you have located the problematic process, the metadata can be regenerated and synchronization occurs as expected.

Reviewer: ```suggestion
```

Rationale: once packages are unlocked on clients attempts to update it will not fail any more. Metadata regeneration has nothing to do with this.

> .Procedure: Resolving Package Inconsistencies

Reviewer: ```suggestion
.Procedure: Removing Package Locks
```

. On the client, check package locks and exclude lists to determine if packages are locked or excluded:
+

> * On an Expanded Support Platform, check [path]``/etc/yum.conf`` and search for ``exclude=``.

Reviewer: Add openSUSE here (works like SLE)
